java/lang/security/audit/el-injection.yaml

```diff
@@ -1,9 +1,9 @@
 rules:
 - id: el-injection
   metadata:
-    - cwe: "CWE-94: Improper Control of Generation of Code ('Code Injection')"
-    - owasp: "A1: Injection"
-    - source-rule-url: https://find-sec-bugs.github.io/bugs.htm#EL_INJECTION
```

Comment:
I noticed the semgrep.live PR-s have this, we should maybe still support `-` operator until those PR-s adhere to this.

Reply:
Maybe we can make a future PR for it when it becomes a problem. I think ultimately @dlukeomalley was right about standardizing on just numbers and converting to the full representation when consumed.